scRNA-seq/05-clustering_markers_scRNA.Rmd

```diff
@@ -249,20 +249,25 @@ purrr::map(
 
 This returns a list of data frames, which isn't quite what we want.
 
-There is no built-in function that will give us just the first few _row names_, so we will have to define one, and `purrr` gives us a nice shorthand for doing that with `~` syntax.
-In this syntax we type a `~` followed by an expression (R code) that uses `.x` as a placeholder for a single element of the list.
-`purrr::map()` will then apply the expression to each element of the list, and return the results as a new list.
+There is no built-in function that will give us just the first few _row names_, so we will have to define one.
+As of version 4.1, R introduced a new approach to defining _anonymous functions_ - that is, functions you can quickly define "on-the-fly" without formally assigning them.
```

Comment:
Ending the sentence with "formally assigning them" seemed odd. I don't like my solution either, tbh, but it is maybe something. 
```suggestion
As of version 4.1, R introduced a new approach to defining _anonymous functions_ - that is, functions you can quickly define "on-the-fly" without formally assigning them to a function name.
```